addons/purchase/data/mail_template_data.xml

```diff
@@ -23,6 +23,10 @@
         If you have any questions, please do not hesitate to contact us.
         <br/><br/>
         Best regards,
+        <t t-if="not is_html_empty(object.user_id.signature)">
+            <br />
+            <t t-out="object.user_id.signature or ''">--<br/>Mitchell Admin</t>
```

Comment:
The ` or ''` part seems unnecessary since we just checked that there is in fact a signature.

Reply:
Actually not in editor mode I think, where qweb is "raw but with placeholders". I think you may end up with Falsy values displayed instead. To be honest: did not test, just copy pasted what Nby did.